roles/lib_utils/action_plugins/sanity_checks.py

```diff
@@ -0,0 +1,123 @@
+"""
+Ansible action plugin to ensure inventory variables are set
+appropriately and no conflicting options have been provided.
+"""
+from ansible.plugins.action import ActionBase
+from ansible import errors
+
+# Valid values for openshift_deployment_type
+VALID_DEPLOYMENT_TYPES = ('origin', 'openshift-enterprise')
+
+# Tuple of variable names and default values if undefined.
+NET_PLUGIN_LIST = (('openshift_use_openshift_sdn', True),
+                   ('openshift_use_flannel', False),
+                   ('openshift_use_nuage', False),
+                   ('openshift_use_contiv', False),
+                   ('openshift_use_calico', False))
+
+
+def to_bool(var_to_check):
+    """Determine a boolean value given the multiple
+       ways bools can be specified in ansible."""
+    yes_list = (True, 1, "True", "1", "true", "Yes", "yes")
```

Comment:
I think "on" and "On" and others are missing from this list.  'y' and 'Y' are also in the list.  

http://yaml.org/type/bool.html

Reply:
Nice.  Had no idea on was one.  Need to include a couple more varients as well.